purchase_order_approval_block/models/purchase_order.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import _, api, fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    approval_block_id = fields.Many2one(
+        comodel_name="purchase.approval.block.reason", string="Approval Block Reason",
+    )
+    approval_blocked = fields.Boolean(
+        "Approval Blocked", compute="_compute_approval_blocked",
+    )
+
+    def _compute_approval_blocked(self):
+        for rec in self:
+            if rec.approval_block_id:
+                rec.approval_blocked = True
```

Comment:
Compute fields on version 13.0 must always return a value, add an equal False if no approval_block_id is found.

Reply:
Thanks, comment attended